### DISCUSSION TITLE
Using Filament for both admin and "frontend"

### DISCUSSION THREAD

**Discussion by oliverbj** [filamentphp/filament#9620] [Help]
Hello Filament Community,

I am currently developing a Laravel application and am considering using Filament for both the user-facing frontend and the admin backend. I have a specific structure in mind:

The root path (/) for the main user interface, which includes standard features like user login, registration, etc.
A separate admin path (/admin) exclusively for administrative functions.

**My main questions are:**

1. Is it feasible to use Filament for the frontend in a way that is seamless with the admin backend, especially considering that they have different layouts and functionalities?
2. How can I effectively manage routing and middleware to ensure that the admin routes are secure and only accessible to authenticated administrators?
3. Are there any best practices for using Filament components on the frontend, given that it's primarily designed for admin panels?
4. Can the same Laravel authentication system be efficiently used for both the frontend and the admin backend with different roles or permissions?

I'm looking for advice or insights from anyone who has experience in setting up a similar structure or from those familiar with the intricacies of Filament in a dual-role setup like this. Any guidance, tips, or resources you can provide would be greatly appreciated.

Thank you in advance for your time and help!

**Comment by sinnbeck** [comment]
1. Yes. I do this. The frontend is using filament forms etc. and a custom layout that fits what I need there. The panel/admin is just the default filament panel
2. That is a built in feature. https://filamentphp.com/docs/3.x/panels/users#authorizing-access-to-the-panel
3. Just follow the guide. Thats what I did and it works great https://filamentphp.com/docs/3.x/forms/adding-a-form-to-a-livewire-component
4. Yeah I personally use fortify but you can also just use what comes with filament. That will still sign in a regular laravel user. https://filamentphp.com/docs/3.x/panels/users#authentication-features

**Reply by coolsam726** [reply]
I personally use two different Filament Panels, one for the Admin Panel and another one for the Customer-facing portal. For the customer-facing portal I use `->topNavigation()` on the panel. However, if you want more control on the layout then I guess you can use the individual filament packages (forms, tables etc) individually in Livewire Components.
Here is a glimpse of my portal:
![image](https://github.com/filamentphp/filament/assets/5610289/78e0a5c0-dda7-491f-aebe-a87393ce8668)
